doc/sphinx/source/inspection/index.rst

```diff
@@ -0,0 +1,35 @@
+.. inspection/index: 
+
+Visualization Tools
+###################
+
+nGraph provides serialization and deserialization facilities along with the 
+ability to create image formats. When visualization is enabled, a ``dot`` file 
+is generated, along with a ``png``. The default can be adjusted by setting the 
+``NGRAPH_VISUALIZE_TREE_OUTPUT_FORMAT`` flag to another format, like PDF. It
+can then be inspected with a third-party visualization tool such as:
+
+#. `Netron`_
```

Comment:
Netron is not supported, at least not yet, and it wouldn't be operating on the `dot` or `png`/`pdf` files.

Reply:
I commented this out for now with the note that it is tentatively planned.